### PR TITLE
Release 224.0.0

### PR DESCRIPTION
**Pull request by matteoscurati**
## Explanation

This is a RC for v224.0.0. See changelog for more details

@metamask/notification-services-controller:0.10.1

## References

- https://consensyssoftware.atlassian.net/browse/NOTIFY-1230
- https://consensyssoftware.atlassian.net/browse/NOTIFY-1238

## Changelog

```
## [0.11.0]

### Added

- Added support for an optional FCM token parameter for push notifications on mobile platforms, allowing native handling of FCM token creation through the Firebase SDK ([#4823](https://github.com/MetaMask/core/pull/4823))

### Changed

- update the types described in `types/on-chain-notification/schema` and `types/on-chain-notification/on-chain-notification` ([#4818](https://github.com/MetaMask/core/pull/4818))

  - adds new notifications: aave_v3_health_factor; ens_expiration; lido_staking_rewards; notional_loan_expiration; rocketpool_staking_rewards; spark_fi_health_factor
  - splits Wallet Notifications from Web 3 Notifications

- updated and added new notification mocks ([#4818](https://github.com/MetaMask/core/pull/4818))
  - can be accessed through `@metamask/notification-services-controller/notification-services/mocks`

### Fixed

- made `updateMetamaskNotificationsList` function work correctly by making the message handler async and moving the publish call outside of the update function. This ensures the `NotificationServicesController:notificationsListUpdated` event is received by the extension ([#4826](https://github.com/MetaMask/core/pull/4826))
```

## Checklist

- [x] I've updated the test suite for new or updated code as appropriate
- [x] I've updated documentation (JSDoc, Markdown, etc.) for new or updated code as appropriate
- [x] I've highlighted breaking changes using the "BREAKING" category above as appropriate
- [x] I've prepared draft pull requests for clients and consumer packages to resolve any breaking changes

### CODE REVIEW

**Review by Prithpal-Sooriya — APPROVED** [review]

**Review by Prithpal-Sooriya — CHANGES_REQUESTED** [review]
wait check the versions!

**Review by Prithpal-Sooriya — APPROVED** [review]

File: packages/notification-services-controller/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.10.1]
+
+### Changed
+
+- update the types described in `types/on-chain-notification/schema` and `types/on-chain-notification/on-chain-notification` ([#4818](https://github.com/MetaMask/core/pull/4818))
+- update the mock data according to the new types in `mock-raw-notifications` ([#4818](https://github.com/MetaMask/core/pull/4818))
```
Comment (Gudahtt) [inline packages/notification-services-controller/CHANGELOG.md:15]: This appears to be talking about test data that is not published with the package. These changelogs are written for people using this package, not for contributors _to_ the package, so things like this should be omitted

```suggestion
```

File: packages/notification-services-controller/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.10.1]
+
+### Changed
+
+- update the types described in `types/on-chain-notification/schema` and `types/on-chain-notification/on-chain-notification` ([#4818](https://github.com/MetaMask/core/pull/4818))
```
Comment (Gudahtt) [inline packages/notification-services-controller/CHANGELOG.md:14]: Could you describe the nature of these changes? How do these types affect users of this package?

It's not relevant here which internal types are changed. We should be describing how these changes impact someone who uses this package. Are these types part of the public API of the package? Are these breaking changes?

File: packages/notification-services-controller/CHANGELOG.md (hunk 1 above)
Comment (Prithpal-Sooriya) [reply packages/notification-services-controller/CHANGELOG.md:15]: These mocks are accessible and used on clients.

We export these and can be used via subpath exports/imports.

```
import { ... } from '@metamask/notification-services-controller/notification-services/mocks`
```

File: packages/notification-services-controller/CHANGELOG.md (hunk 2 above)
Comment (Prithpal-Sooriya) [reply packages/notification-services-controller/CHANGELOG.md:14]: Agreed, these types/schema changes (non-breaking) are important. They introduce new notifications that we are adding onto extension and mobile.

Comment (Prithpal-Sooriya) [reply packages/notification-services-controller/CHANGELOG.md:14]: @matteoscurati can you update the changelog to represent these in the next release (tomorrow)

Comment (Prithpal-Sooriya) [reply packages/notification-services-controller/CHANGELOG.md:14]: @matteoscurati can you update the changelog to be more descriptive on this change?

File: packages/notification-services-controller/CHANGELOG.md (hunk 1 above)
Comment (Gudahtt) [reply packages/notification-services-controller/CHANGELOG.md:15]: Oh I see, interesting, nevermind then!

File: packages/notification-services-controller/CHANGELOG.md (hunk 2 above)
Comment (matteoscurati) [reply packages/notification-services-controller/CHANGELOG.md]: @Gudahtt @Prithpal-Sooriya thanks for the comment. You’re right. I’ve improved the description to better explain the non-breaking changes implemented. Can I ask you for a new review?

**Review by Prithpal-Sooriya — APPROVED** [review]

**Pull request merged**